index.bs

```diff
@@ -465,7 +465,7 @@ To <dfn>pullDatagrams</dfn>, given a {{WebTransport}} object |transport|, run th
     1. Set |chunk| to |view|.
 1. Otherwise:
   1. Set |offset| to 0.
-  1. Set |maxBytes| to an [=implementation-defined=] size.
+  1. Set |maxBytes| to the size of |bytes|.
```

Comment:
I think this can actually just be
```
1. Set |maxBytes| to |bytes|.
```

Reply:
Sorry, I was wrong. The original text was correct.

Reply:
I think this suggests the name `bytes` is confusing here now that we're doing BYOB. [[[IncomingDatagramsQueue]]](https://pr-preview.s3.amazonaws.com/nidhijaju/webtransport/pull/487.html#dom-webtransportdatagramduplexstream-incomingdatagramsqueue-slot) says it's a "datagram" which might be a better term to help separate inputs from outputs, or even "incomingDatagram"?